web/modules/custom/dpl_fees/src/Form/FeesListSettingsForm.php

```diff
@@ -63,28 +63,43 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
 
     $form['settings'] = [
       '#type' => 'fieldset',
-      '#title' => $this->t('Basic settings'),
+      '#title' => $this->t('Basic settings', [], ['context' => 'Fees list settings form']),
       '#tree' => FALSE,
     ];
 
     $form['settings']['fees_and_replacement_costs_url'] = [
-      '#type' => 'url',
-      '#title' => $this->t('Fees and Replacement costs URL'),
-      '#description' => $this->t('File or URL containing the fees and replacement costs'),
+      '#type' => 'linkit',
+      '#title' => $this->t('Fees and Replacement costs URL', [], ['context' => 'Fees list settings form']),
+      '#description' => $this->t('File or URL containing the fees and replacement costs. <br>
+                                         You can add a relative url (e.g. /takster). <br>
+                                         You can search for an internal url. <br>
+                                         You can add an external url (starting with "http://" or "https://").', [], ['context' => 'Fees list settings form']),
+      '#autocomplete_route_name' => 'linkit.autocomplete',
+      '#autocomplete_route_parameters' => [
+        'linkit_profile_id' => 'default',
+      ],
       '#default_value' => $config->get('fees_and_replacement_costs_url') ?? DplFeesSettings::FEES_AND_REPLACEMENT_COSTS_URL,
```

Comment:
This is not a part of the scope, but didnt we create methods for getting fee settings like this?

Reply:
Yes, we have implemented some of these methods, but not throughout the whole project yet. I have added 2 new methods for getting fee settings now.